### ISSUE TITLE
Failed to open pull request, even with github access token defined

### ISSUE THREAD

**Issue by Kukanani** [ros-infrastructure/bloom#632]
I am attempting to bloom `https://github.com/ros-perception/vision_msgs` into `rolling`, but the final pull request into `rosdistro` is failing with the following error:
```
Failed to open pull request: GithubException - HTTP Error 404: Not Found (https://api.github.com/repos/ros/rosdistro/pulls): None
```

Full output is here: [bloom-attempt-stdout.txt](https://github.com/ros-infrastructure/bloom/files/6311270/bloom-attempt-stdout.txt)

Bloom version:
```
$ bloom-release --version
0.10.6
```

Github personal access token:
```
$ cat ~/.config/bloom
{
    "github_user": "kukanani",
    "oauth_token": "ghp_<token_contents>"
}
```

Access token scopes:
![Screenshot from 2021-04-14 08-59-14](https://user-images.githubusercontent.com/6893343/114722832-c0319300-9cff-11eb-8add-ce30a04a1091.png)

Other things to note that may be helpful:
```
$ uname -a
Linux obelisk20 5.8.0-48-generic #54~20.04.1-Ubuntu SMP Sat Mar 20 13:40:25 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```
* Release repo and source repo are under different users (kukanani and ros-perception, respectively)
* Tested multiple times several minutes apart, same result
* I can visit the URL that is supposedly 404ing in Firefox (showing 29 pulls at the moment)

**Comment by Kukanani** [comment]
Just wanted to follow up on this and note that I don't have 2FA on my account.

**Comment by chapulina** [comment]
I'm also having the exact same issue. I have 2FA and SSH setup.

**Comment by ivanpauno** [comment]
Same happening to me https://github.com/ros/rosdistro/pull/29105#issuecomment-818079290.

**Comment by ivanpauno** [comment]
FWIW, I'm also using bloom 0.10.6

**Comment by nuclearsandwich** [comment]
Thanks for the report. I'm going to work on reproducing and investigating the issue.

> Just wanted to follow up on this and note that I don't have 2FA on my account.

**Comment by nuclearsandwich** [comment]
@Kukanani this should be fixed in bloom 0.10.7! Also hi! :wave:

**Comment by Kukanani** [comment]
:wave: :grin: 
Fix verified working, thanks!
